ibis/mapd/operations.py

```diff
@@ -330,9 +330,43 @@ def _cross_join(translator, expr):
     return translator.translate(left.join(right, ibis.literal(True)))
 
 
+def _format_point_value(value):
+    return ' '.join([str(v) for v in value])
```

Comment:
You don't need to construct a `list` when calling `str.join`.

Reply:
thanks!

Reply:
@xmnlab please remove the list comprehension and use a generator instead like Phillip mentioned.

Reply:
sounds good. sorry I lost that for a unknown reason. I am fixing that again. thanks again!